test/clamm.test.ts

```diff
@@ -1534,6 +1559,20 @@ describe('math tests', () => {
       }
       await expectError(clamm.methods.calculateAmountDelta(params))
     })
+
+    test('error handling 2', async () => {
```

Comment:
The test names should be more precise, you should mention which error the cases handle